### DISCUSSION TITLE
Recoil/atom_effects/

### DISCUSSION THREAD

**Discussion by giscus** [yeopto/yeopto.github.io#15] [Comments]
# Recoil/atom_effects/

장바구니 미션에서 Atom Effects 사용기

https://yeopto.github.io/Recoil/atom_effects/

**Comment by ukkodeveloper** [comment]
Atom Effect 정말 편리해 보이네요. 굳이 useEffect를 사용하지 않아도 atom state에 effect를 걸어준다는 게 정말 인상 깊네요. 저도 적용해보고 싶어요!

**Comment by Gilpop8663** [comment]
atomEffect 대체 어떻게 사용하는지 공식 문서를 봐도 잘 몰랐는데 엽토의 글에 잘 정리되어 있네요!

관심사를 분리한다는 점에서 좋은 것 같아요

글 잘 보았습니다 👍

**Comment by kimsuyeon0916** [comment]
타입스크립트로 적용해주신 자료 덕분에 참고하여 기능 완성했습니다! 좋은 글 감사드립니다😊
